test/Altinn.Profile.Tests/IntegrationTests/API/Controllers/UnitContactPointControllerTests.cs

```diff
@@ -93,6 +93,25 @@ public async Task PostLookup_ErrorResult_ReturnsProblemDetails()
             Assert.Equal(HttpStatusCode.InternalServerError, response.StatusCode);
         }
 
+        [Theory]
+        [InlineData("not json")]
```

Comment:
The addition of 
```
        if (!ModelState.IsValid)
        {
            return BadRequest(ModelState);
        }
```
.. in the Controller probably changes the code flow when `UnitContactPointLookup.OrganizationNumbers = null`. I think this would propagate to the repo/SBL-client an HTTP request to SBLBridge before (probably resulting in a `500` response?), while now it's stopped in the model validation (`400` response).

```suggestion
        [InlineData("not json")]
        [InlineData("{\"OrganizationNumbers\":null,\"ResourceId\":null}")]
```

Reply:
Thank you, even the test arrangement were wrong and this gave me the reason to look into it. I'm honestly a bit surprised that model validation does null reference checking of non-nullable references. I thought that had to be done explicitly. In other words; I assumed that your example would result in a different response and not a BadRequest.

Reply:
Would still need to have explicit validation the content of the list. One day.